configure.ac

```diff
@@ -160,33 +160,26 @@ AC_ARG_ENABLE(runtime,
 # Check for enable/disable builtin PSL data
 AC_ARG_ENABLE(builtin,
   [
-  --enable-builtin[[=IDNA library]]
-      Specify the IDNA library used for built-in data generation:
-        libidn2 [[default]]: IDNA2008 library (also needs libunistring)
-        libicu: IDNA2008   UTS#46 library
-        libidn:            IDNA2003 library (also needs libunistring)
+  --enable-builtin         Generate built-in PSL data
   --disable-builtin        Do not generate built-in PSL data
   ], [
-    if test "$enableval" = "libidn2" -o "$enableval" = "yes"; then
-      enable_builtin=libidn2
-      AC_DEFINE([BUILTIN_GENERATOR_LIBIDN2], [1], [generate PSL data using libidn2])
-    elif test "$enableval" = "libicu"; then
-      enable_builtin=libicu
-      AC_DEFINE([BUILTIN_GENERATOR_LIBICU], [1], [generate PSL data using libicu])
-    elif test "$enableval" = "libidn"; then
-      enable_builtin=libidn
-      AC_DEFINE([BUILTIN_GENERATOR_LIBIDN], [1], [generate PSL data using libidn])
+    if test "$enableval" = "yes"; then
+      enable_builtin=yes
     elif test "$enableval" = "no"; then
       enable_builtin=no
     else
       AC_MSG_ERROR(Unknown value $enableval)
     fi
   ], [
     # this is the default if neither --enable-builtin nor --disable-builtin were specified
-    enable_builtin=auto
+    enable_builtin=yes
```

Comment:
I wonder if it makes sense to use `enable_builtin=no`.

The reason is that (from what I experience), downstream maintainers do not update libpsl regularly with actual PSL data. So in most (Linux) distributions, the built-in data bit rots.

The downside is that having no built-in data requires an extra package for PSL that is regularly updated. Something that I also doesn't see.

WDYT ?

Reply:
I think the only case where it would make sense to disable the builtin data would be when `--with-psl-distfile` is used.
Else the default config wouldn't have any data (except an application specific file name, but most clients probably rely on the default data).